pkgs/by-name/pa/payme/package.nix

```
@@ -0,0 +1,23 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "payme";
+  version = "v1.2.0";
+
+  src = fetchFromGitHub {
+    owner = "jovandeginste";
+    repo = pname;
+    rev = version;
+    hash = "sha256-WE/sAs0VSeb5UKkUy1iyjyXtgDmlQhdZkw8HMMSbQiE=";
+  };
+
+  vendorHash = null;
+
+  meta = with lib; {
+    description = "QR code generator (ASCII & PNG) for SEPA payments";
+    homepage = "https://github.com/jovandeginste/payme";
+    license = licenses.mit;
+    maintainers = with maintainers; [ cimm ];
+    platforms = with lib.platforms; unix;
```

```suggestion
    license = lib.licenses.mit;
    maintainers = with lib.maintainers; [ cimm ];
```

`platforms` is unneeded here - since it use nothing besides Go